ChangeLog

```diff
@@ -1,3 +1,33 @@
+Jan 20, 2024: version 7.5.2
+
+    * CHOLMOD 5.1.2: bug fix (restore ABI compatibility with 5.0.x)
```

Comment:
Could it be made clearer that this is an ABI-breaking change with respect to CHOLMOD 5.1.X?
``` suggestion
    * CHOLMOD 5.1.2: bug fix (restore ABI compatibility with 5.0.x, i.e., 5.1.2 is ABI incompatible to 5.1.x)
```
This sound self-contradicting. But that sentence would make more sense if the upcoming version would be CHOLMOD 5.2.0 instead.

Reply:
Yes -- that does make more sense.  I'll do CHOLMOD 5.2.0 and SuiteSparse 7.6.0.